src/components/auth-code/auth-code-controller.ts

```diff
@@ -24,8 +24,14 @@ export function authCodeGet(
       req.session.client,
       req.session.user
     );
+
     req.session.user.authCodeReturnToRP = false;
 
+    // Remove the reauthenticate property regardless of its value
```

Comment:
So we tend not to comment in the code like this, I tend to prefer to put this in the commit message. I think we should probably have a team chat about our approach to comments at some point, so this one non-blocking, I'll let you decide whether you want to keep it or boost the commit message.